java.lang.ClassNotFoundException with version 1.0.3

## Description
Deployment of version 1.0.3 fails with 
`java.lang.NoClassDefFoundError: com/google/common/collect/ImmutableSet`
Deployment of version 1.0.2 in the same environment works just fine.

```
12:48:22,305 INFO  [org.jboss.as.server.deployment] (MSC service thread 1-8) WFLYSRV0027: Starting deployment of "keycloak-metrics-spi-1.0.3.jar" (runtime-name: "keycloak-metrics-spi-1.0.3.jar")
12:48:22,390 INFO  [org.keycloak.subsystem.server.extension.KeycloakProviderDeploymentProcessor] (MSC service thread 1-6) Deploying Keycloak provider: keycloak-metrics-spi-1.0.3.jar
12:48:22,402 ERROR [org.jboss.msc.service.fail] (MSC service thread 1-6) MSC000001: Failed to start service jboss.deployment.unit."keycloak-metrics-spi-1.0.3.jar".POST_MODULE: org.jboss.msc.service.StartException in service jboss.deployment.unit."keycloak-metrics-spi-1.0.3.jar".POST_MODULE: WFLYSRV0153: Failed to process phase POST_MODULE of deployment "keycloak-metrics-spi-1.0.3.jar"
	at org.jboss.as.server.deployment.DeploymentUnitPhaseService.start(DeploymentUnitPhaseService.java:183)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.startService(ServiceControllerImpl.java:1738)
	at org.jboss.msc.service.ServiceControllerImpl$StartTask.execute(ServiceControllerImpl.java:1700)
	at org.jboss.msc.service.ServiceControllerImpl$ControllerTask.run(ServiceControllerImpl.java:1558)
	at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
	at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:1982)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1486)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1377)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.NoClassDefFoundError: com/google/common/collect/ImmutableSet
	at org.jboss.aerogear.keycloak.metrics.MetricsFilter.<clinit>(MetricsFilter.java:20)
	at org.jboss.aerogear.keycloak.metrics.MetricsEndpointFactory.init(MetricsEndpointFactory.java:19)
	at org.keycloak.services.DefaultKeycloakSessionFactory.loadFactories(DefaultKeycloakSessionFactory.java:217)
	at org.keycloak.services.DefaultKeycloakSessionFactory.deploy(DefaultKeycloakSessionFactory.java:115)
	at org.keycloak.provider.ProviderManagerRegistry.deploy(ProviderManagerRegistry.java:42)
	at org.keycloak.subsystem.server.extension.KeycloakProviderDeploymentProcessor.deploy(KeycloakProviderDeploymentProcessor.java:55)
	at org.jboss.as.server.deployment.DeploymentUnitPhaseService.start(DeploymentUnitPhaseService.java:176)
	... 8 more
Caused by: java.lang.ClassNotFoundException: com.google.common.collect.ImmutableSet from [Module "deployment.keycloak-metrics-spi-1.0.3.jar" from Service Module Loader]
	at org.jboss.modules.ModuleClassLoader.findClass(ModuleClassLoader.java:255)
	at org.jboss.modules.ConcurrentClassLoader.performLoadClassUnchecked(ConcurrentClassLoader.java:410)
	at org.jboss.modules.ConcurrentClassLoader.performLoadClass(ConcurrentClassLoader.java:398)
	at org.jboss.modules.ConcurrentClassLoader.loadClass(ConcurrentClassLoader.java:116)
	... 15 more

12:48:22,410 ERROR [org.jboss.as.controller.management-operation] (DeploymentScanner-threads - 1) WFLYCTL0013: Operation ("deploy") failed - address: ([("deployment" => "keycloak-metrics-spi-1.0.3.jar")]) - failure description: {"WFLYCTL0080: Failed services" => {"jboss.deployment.unit.\"keycloak-metrics-spi-1.0.3.jar\".POST_MODULE" => "WFLYSRV0153: Failed to process phase POST_MODULE of deployment \"keycloak-metrics-spi-1.0.3.jar\"
    Caused by: java.lang.NoClassDefFoundError: com/google/common/collect/ImmutableSet
    Caused by: java.lang.ClassNotFoundException: com.google.common.collect.ImmutableSet from [Module \"deployment.keycloak-metrics-spi-1.0.3.jar\" from Service Module Loader]"}}
12:48:22,476 INFO  [org.jboss.as.server] (DeploymentScanner-threads - 1) WFLYSRV0010: Deployed "keycloak-metrics-spi-1.0.3.jar" (runtime-name : "keycloak-metrics-spi-1.0.3.jar")
12:48:22,477 INFO  [org.jboss.as.controller] (DeploymentScanner-threads - 1) WFLYCTL0183: Service status report
WFLYCTL0186:   Services which failed to start:      service jboss.deployment.unit."keycloak-metrics-spi-1.0.3.jar".POST_MODULE: WFLYSRV0153: Failed to process phase POST_MODULE of deployment "keycloak-metrics-spi-1.0.3.jar"

```


## Environment

* Operating system: RHEL 7
* Java versions: java-1.8.0-openjdk-1.8.0.212.b04-0.el7_6.x86_64

## Steps to reproduce

- Download and extract Keycloak archive
- Start Keycloak
- `cp keycloak-metrics-spi-1.0.3.jar keycloak-6.0.1/standalone/deployments`

--- a/src/main/java/org/jboss/aerogear/keycloak/metrics/MetricsEventListenerSpi.java
+++ b/src/main/java/org/jboss/aerogear/keycloak/metrics/MetricsEventListenerSpi.java
@@ -4,8 +4,6 @@ import org.keycloak.events.EventListenerSpi;
 import org.keycloak.provider.Provider;
 import org.keycloak.provider.ProviderFactory;
 
-import java.util.EventListener;
-
 public class MetricsEventListenerSpi extends EventListenerSpi {
 
     @Override

--- a/src/main/java/org/jboss/aerogear/keycloak/metrics/MetricsFilter.java
+++ b/src/main/java/org/jboss/aerogear/keycloak/metrics/MetricsFilter.java
@@ -1,14 +1,15 @@
 package org.jboss.aerogear.keycloak.metrics;
 
-import com.google.common.collect.ImmutableSet;
+import org.jboss.logging.Logger;
 
 import javax.ws.rs.container.ContainerRequestContext;
 import javax.ws.rs.container.ContainerRequestFilter;
 import javax.ws.rs.container.ContainerResponseContext;
 import javax.ws.rs.container.ContainerResponseFilter;
 import javax.ws.rs.core.MediaType;
+import java.util.Collections;
+import java.util.HashSet;
 import java.util.Set;
-import org.jboss.logging.Logger;
 
 public final class MetricsFilter implements ContainerRequestFilter, ContainerResponseFilter {
     private static final Logger LOG = Logger.getLogger(MetricsFilter.class);
@@ -17,11 +18,13 @@ public final class MetricsFilter implements ContainerRequestFilter, ContainerRes
     private static final MetricsFilter INSTANCE = new MetricsFilter();
 
     // relevant response content types to be measured
-    private static final Set<MediaType> CONTENT_TYPES = ImmutableSet.of(
-        MediaType.APPLICATION_JSON_TYPE,
-        MediaType.TEXT_HTML_TYPE,
-        MediaType.APPLICATION_XML_TYPE
-    );
+    private static final Set<MediaType> contentTypes = new HashSet<>();
+    static {
+        contentTypes.add(MediaType.APPLICATION_JSON_TYPE);
+        contentTypes.add(MediaType.APPLICATION_XML_TYPE);
+        contentTypes.add(MediaType.TEXT_HTML_TYPE);
+    }
+    private static final Set<MediaType> CONTENT_TYPES = Collections.unmodifiableSet(contentTypes);
 
     public static MetricsFilter instance() {
         return INSTANCE;

